packagers/pnpm-workspaces-custom-output/package.json

```diff
@@ -9,7 +9,8 @@
   },
   "scripts": {
     "cmd": "ts-node ./script.ts",
-    "generate": "prisma generate"
+    "generate": "prisma generate",
+    "v": "prisma -v"
```

Comment:
This is weird, but follows the pattern already in place. Maybe something to clean up in a follow up PR.

Reply:
Yes, version is a name already reserved by NPM, so I couldn't use it.

Reply:
I actually meant that this project uses npm scripts instead of calling the commands directly - not sure why that is necessary.